validation-test/compiler_scale/bind_extension_decl.gyb

```diff
@@ -0,0 +1,6 @@
+// RUN: %scale-test --sum-multi --typecheck --begin 5 --end 16 --step 5 --select validateDecl %s
+// REQUIRES: OS=macosx
+// REQUIRES: asserts
```

Comment:
Why does this require assertions?

Reply:
Probably residue from when scale-test needed assertions (because it used LLVM's stats reporting). It now works using stats-output-dir when available too, so long as you're measuring one of the always-available stats.